src/langs/fr.json

```diff
@@ -0,0 +1,9 @@
+{
+  "translations": {
+    "New App Data": "Créer une nouvelle donnée de l'App",
+    "Update Setting": "Mettre à jour le réglage",
+    "Local Context": "Contexte local",
+    "App Context": "Contexte de l'App",
+    "Members": "Membres"
```

Comment:
Adapt translations. You can do that later if you want (then turn this comment into an issue).